students/yushus/session03/mailroom.py

```diff
@@ -0,0 +1,103 @@
+import sys  # imports go at the top of the file
+import random
+
+donor_db = [("William Gates, III", [653772.32, 12.17]),
+            ("Jeff Bezos", [877.33]),
+            ("Paul Allen", [663.23, 43.87, 1.32]),
+            ("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]),
+            ("Elon Musk", [250000, 15000])
+            ]
+
+prompt = "\n".join(("Welcome to mailroom bot!",
+          "Please choose from below options:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - quit",
+          ">>> "))
+
+# sort by the total amount of donated
+def sort_key(donor):
+    return sum(donor[1])
+
+def print_donor():
+    donors = [x[0] for x in donor_db]
+    donors_list = "\n".join(donors)
+    print(donors_list)
+
+def print_email(name, amount):
+    letter = '''
+        "Dear {name},
```

Comment:
Minor: Did you want the leading double quote char? Also, there'll be leading spaces on every line, and not sure if that's what you wanted? In fact, that's kind of a common situation, so I remember there's a Python package that strips the leading spaces in the `'''...'''` strings, but I forgot the name... Maybe `dedent`? You may try to look it up... Not a major issue, though.

Reply:
Thank you so much Hosung! Yeah, the leading double quote is a mistake and will remove that. My initial thought is to make the email content itself "stand out" from the rest, hence chose to use ' ' '